Makefile

```
@@ -14,6 +14,8 @@ test-compile:
 		  --retain-lines -d ./tests-compiled --out-file-extension=.mjs
 
 test: test-compile
```

we should add the new target here:

```suggestion
test: test-compile test-mocha
```